components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/sharing/AccountSharing.kt

```diff
@@ -172,6 +173,7 @@ object AccountSharing {
      * but not signature rotation.
      * @return A certificate SHA256 fingerprint, if one could be reliably obtained.
      */
+    @SuppressLint("PackageManagerGetSignatures")
     fun getSignaturePreAPI28(packageManager: PackageManager, packageName: String): String? {
```

Comment:
This code looks similar to what we have in `AndroidAssetFinder` maybe we should file a issue to reuse and I have one.

Reply:
Yeah, although, afaik in a discussion with @grigoryk we said that we could get rid of this. I'll file something for that.

Reply:
https://github.com/mozilla-mobile/android-components/issues/8719